api/v1alpha1/fenceagentsremediationtemplate_webhook.go

```diff
@@ -37,8 +39,24 @@ func (r *FenceAgentsRemediationTemplate) SetupWebhookWithManager(mgr ctrl.Manage
 
 // TODO(user): EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 
+// +kubebuilder:webhook:path=/mutate-fence-agents-remediation-medik8s-io-v1alpha1-fenceagentsremediationtemplate,mutating=true,failurePolicy=fail,sideEffects=None,groups=fence-agents-remediation.medik8s.io,resources=fenceagentsremediationtemplates,verbs=create;update,versions=v1alpha1,name=mfenceagentsremediationtemplate.kb.io,admissionReviewVersions=v1
```

Comment:
Does the following check happen on creation and update or only for creation? I am asking due to the verbs that are used in this line `verbs=create;update`

Reply:
IIUC both.
I've done a test trying to remove the annotation and the update webhook triggering kept adding it.